commit 1da3c6d10720e883f92b2d758cc61a470361a8de (refs/changes/61/261/1)
Author: Todd Whiteman <todd.whiteman@joyent.com>
Date:   2016-08-09T14:48:53-07:00 (3 years, 2 months ago)
    
    DOCKER-908: docker build: cn-agent - remove test files from npm install of sdc-docker-build

diff --git a/.npmignore b/.npmignore
new file mode 100644
index 0000000..be6b084
--- /dev/null
+++ b/.npmignore
@@ -0,0 +1,6 @@
+.npmignore
+deps
+node_modules
+test
+tools
+Makefile
